docs/en/setup/backend/backend-health-check.md

```diff
@@ -56,6 +56,12 @@ If some modules are unhealthy (e.g. storage H2 is down), then the result may loo
 Refer to [checkHealth query](https://github.com/apache/skywalking-query-protocol/blob/master/common.graphqls)
 for more details.
 
+In some cases, GraphQL query may not be a good choice to check the health status of OAP, for example, in Kubernetes
```

Comment:
As we are running on k8s in most cases, we could move this as the first part of health check doc, rather than `in some cases` only.

Should we link to your new API doc from here? 

One more, your new API doc doesn't have a menu item, no one could find that doc.